src/Login.jsx

```diff
@@ -1,23 +1,35 @@
 import React, { useState } from "react"
+import { Welcome } from './Welcome'
 
-export const Login = () => {
+export const Login = (props) => {
     const [email, setEmail ] = useState('');
     const [pass, setPass ] = useState('');
+    const [page, setPage] = useState(false)
 
-    const handleSubmit = () => {
-        email.preventDefault();
-        console.log(email);
+
+    const handleSubmit = (e) => {
+        e.preventDefault(); // Prevent default form submission behavior
+        setPage(true);
+    // setPass()
+    // setEmail()
     }
+    
     return (
-        <>
-            <form onSubmit={handleSubmit}>
-                <label for="email">email </label>
-                <input value={email} type="email" placeholder="youremail@gmail.com" name="email"/>
-                <label for="password">password </label>
-                <input value={pass} type="password" placeholder="*****" name="password" />
-                <button>Log In</button>
-            </form>   
-            <button> Already have an account? Register Here </button>
-        </>
-            )
+        <div>
+            {!page && 
+                <div className="auth-form-container">
+                    <h2> Login </h2>
+                    <form className="login-form" onSubmit={handleSubmit}>
+                        <label htmlFor="email">Email </label>
+                        <input value={email} type="email" placeholder="youremail@gmail.com" name="email"/>
+                        <label htmlFor="password">Password </label>
+                        <input value={pass} type="password" placeholder="*****" name="password" />
+                        <button className='login-btn'>Log In</button>
+                    </form>   
+                    <button className='register-btn' onClick={() => props.onFormSwitch('register')}>Register Here </button>
```

Comment:
break this handler out into it's own method

Reply:
made this change in the code, i believe.